monkey/monkey_island/cc/services/attack/technique_reports/T1065.py

```diff
@@ -14,6 +14,9 @@ class T1065(AttackTechnique):
 
     @staticmethod
     def get_report_data():
-        port = ConfigService.get_config_value(CURRENT_SERVER_PATH).split(":")[1]
+        monkey = Monkey.objects.first()
+        tunnel = monkey.get_tunnel_info()["tunnel"]
```

Comment:
This will not work in general. Tunnel is another monkey: 
![image](https://user-images.githubusercontent.com/36815064/173857673-9980b953-eeb4-4fd3-beeb-ba742a0769cc.png)

Reply:
I'm fixing this now